### DISCUSSION TITLE
怎么接收短信

### DISCUSSION THREAD

**Discussion by ArkWhit** [istoreos/istoreos#1453] [Q&A]
设备：H69K
5G模块：FM160-CN

有什么方法让设备接收短信，然后转换成可供人阅读的文字。
如果可以的话，再转发到邮箱。

**Comment by REviews009** [comment]
你的想法很好，如果有这个功能的话！

**Comment by w9315273** [comment]
如果有这种奇奇怪怪的极其罕见需求的话 最好的办法就是你自己开发了

**Reply by fredzhu-greendot** [reply]
这功能不奇怪啊。 海外手机卡在国内不能漫游，验证码就需要接收短信转邮箱或调个web 服务通知

**Comment by jjm2473** [comment]
istore有个“移动通信模块”的插件，看看里面的功能有没有用
